dfusion_rust_core/src/models/standing_order.rs

```diff
@@ -69,35 +110,214 @@ impl From<mongodb::ordered::OrderedDocument> for StandingOrder {
     }
 }
 
+impl From<Arc<Log>> for StandingOrder {
+    fn from(log: Arc<Log>) -> Self {
+        let mut bytes: Vec<u8> = log.data.0.clone();
+        println!("Parsing StandingOrder from bytes. {} bytes. {:?}", bytes.len(), bytes);
+        
+        // Get basic data from event
+        let batch_index = U256::pop_from_log_data(&mut bytes);
+        let valid_from_auction_id = U256::pop_from_log_data(&mut bytes);
+        let account_id = u16::pop_from_log_data(&mut bytes);
+
+        let bytes_init = u8::pop_from_log_data(&mut bytes) as usize;
+        let byte_size = u8::pop_from_log_data(&mut bytes) as usize;
+
+        println!("Extracting packed order. Bytes: {}-{}", bytes_init, bytes_init + byte_size);
+        let packed_orders_bytes = &bytes[0..byte_size];
+        println!("Parsing orders from packedOrders. {} bytes. {:?}", packed_orders_bytes.len(), packed_orders_bytes);
+        assert!(packed_orders_bytes.len() % 26 == 0, "Each order should be packed in 26 bytes");
+                
+        // Extract packed order info
+        let orders: Vec<models::Order> = packed_orders_bytes.chunks(26)
+            .enumerate()
+            .map(|(order_number, chunk)| models::Order::from(EncodedOrder {
+                account_id,
+                auction_id: batch_index,
```

Comment:
This is not really correct. `batch_index` is the auction_id from which point onwards the standing order is valid. However, the way we query for auction_id is via exact match (where `auction_id == x`). A standing order would therefore only appear in the batch it was created in (but it should in all following ones until it was updated).

Check [here](https://github.com/gnosis/dex-services/blob/2143f1f808411528f976c91082509dc4cdc23af3/event_listener/dfusion_db/database_interface.py#L153) how we combine orders and standing orders in the current python code.

I think the Order entities on a StandingOrderBatch should not have an auction ID on them (at least not when they are written to an entity). This way they are ignored on the one-off order fetching (where auction_id==x) and can be separately fetched.